File: fizzbuzz.rb
#!/usr/bin/ruby

# Language:       Ruby
# Web site:       http://www.ruby-lang.org/
# Last tested on: Ubuntu 20.04
# Requires:       apt-get install ruby

(1..100).each do |i|
    puts "#{if i % 15 == 0
                "FizzBuzz"
            elsif i % 3 == 0
                "Fizz"
            elsif i % 5 == 0
                "Buzz"
            else
                i
            end}"
end

File: fizzbuzz.erl
#!/usr/bin/escript

% Language:       Erlang
% Web site:       http://www.erlang.org/
% Last tested on: Ubuntu 16.10
% Requires:       apt-get install erlang

print_line(N) when N rem 15 == 0 ->
    io:format("FizzBuzz\n");
print_line(N) when N rem 3 == 0 ->
    io:format("Fizz\n");
print_line(N) when N rem 5 == 0 ->
    io:format("Buzz\n");
print_line(N) ->
    io:format("~w\n", [N]).

print_lines(X, Y) when Y >= X ->
    print_line(X),
    print_lines(X+1, Y);
print_lines(_, _) ->
    done.

main(_) ->
    print_lines(1, 100).
